.github/workflows/ci.yml

```diff
@@ -1,6 +1,37 @@
 name: CI
-on: push
+on:
+  push:
+    branches:
+      - '*'
+    tags-ignore:
+      - '*'
 jobs:
+  mix_format:
+    name: mix format (Elixir 1.10.2 OTP 22.3)
+    runs-on: ubuntu-16.04
+    steps:
+      - uses: actions/checkout@v1
+      - uses: actions/setup-elixir@v1.0.0
+        with:
+          otp-version: "22.3"
+          elixir-version: "1.10.2"
+      - name: Install Dependencies
+        run: mix deps.get
+      - name: Run Formatter
+        run: mix format --check-formatted
+  mix_credo:
+    name: mix credo (Elixir 1.10.2 OTP 22.3)
+    runs-on: ubuntu-16.04
+    steps:
+      - uses: actions/checkout@v1
+      - uses: actions/setup-elixir@v1.0.0
+        with:
+          otp-version: "22.3"
+          elixir-version: "1.10.2"
+      - name: Install Dependencies
+        run: mix deps.get
+      - name: Run Credo
+        run: mix credo
```

Comment:
Why not?
```suggestion
        run: mix credo --strict
```

Reply:
@brosquinha It is already setted in the config file :)